assets/javascripts/ng-app/controllers/car_select.js

```diff
@@ -3,6 +3,8 @@ angular.module('myApp')
   ['$scope', 'carSelectService', 'navigationService',
   function ($scope, carSelectService, navigationService) {
 
+  toggleWaitIcon();
```

Comment:
The name for this is a bit confusing since it doesn't actually toggle the loading gif. Also, `waitIcon` is a bit confusing as it's a noun and it's used as a boolean in the code. A better name would be `isLoading`.